docs/src/main/paradox/errorhandling.md

```diff
@@ -28,6 +28,11 @@ Scala
 Java
 : @@ snip [stopping](../../test/java/sample/javadsl/StreamWrapperActor.java) { #errorHandlingSupervisor }
 
+In order to handle **Consumer fails due to WakeupExceptions** while using the ClosedGraph.
```

Comment:
Yeah, I agree, however, context is already set as 'Restarting the stream' :). This is special case of it and it is about closed graph stream scenario!